veracode/utils.py

```diff
@@ -393,3 +401,41 @@ def is_valid_netloc(*, netloc: str) -> bool:
         return True
 
     return False
+
+
+def get_app_id(app_name: str) -> Union[int, None]:
+    """
+    Query for and return the app_id associated with the app_name
+
+    https://help.veracode.com/reader/orRWez4I0tnZNaA_i0zn9g/Z4Ecf1fw7868vYPVgkglww
+    """
+    try:
+        ## Use the setter to apply a default to ensure it is valid
+        version = {"getapplist.do": "5.0"}
```

Comment:
I don't want two upload api version dicts; can we abstract this into `constants.py` for both this and `UploadAPI`'s `self.version`?  Perhaps for consistency we also move the `SandboxAPI` and `ResultsAPI` `self.version` `dict`s.